stdlib/3/subprocess.pyi

```diff
@@ -250,7 +250,12 @@ STDOUT = ...  # type: int
 if sys.version_info >= (3, 3):
     DEVNULL = ...  # type: int
     class SubprocessError(Exception): ...
-    class TimeoutExpired(SubprocessError): ...
+    class TimeoutExpired(SubprocessError):
+        cmd = ...  # type: List[str]
+        timeout = ...  # type: int
```

Comment:
Couldn't this also be a float? And how sure are you it can never be `None`?

Reply:
from a sourcedive, it looks like this can't be None. All the places where it is set either check for None or look like this:

```self.stderr_thread.join(self._remaining_time(endtime))
if self.stderr_thread.is_alive():
    raise TimeoutExpired(self.args, orig_timeout)
```
Where is_alive() won't be true if the endtime/orig_timeout is None